dart/test/sentry_client_test.dart

```diff
@@ -394,6 +394,25 @@ void main() {
       expect(capturedEvent.fingerprint, eventFingerprint);
       expect(capturedEvent.breadcrumbs, eventCrumbs);
     });
+
+    test('should apply the scope user to null event user fields ', () async {
+      final client = SentryClient(options);
```

Comment:
great opportunity to create a fixture pattern, lets always do it when we're touching test code and it's not using such a pattern that helps us writing more concise tests, also, reusing code